Header header
string hash
int64 value
string tag
int64 timestamp
string message
string address_balance
